microsoft-edge/devtools-guide-chromium/3D-view.md

```diff
@@ -0,0 +1,77 @@
+
+### Z-index color type
+
+These are the different visualizations you can use for the DOM in your canvas. Whether you use it for fun or because they help you visualize the DOM better, we have three different colorways as well as a "background color" setting. The radio buttons allow you to toggle through the options and pick the color type most appropriate for your project (or that you like the most).
+
+![DOM view](./media/facebook-dom-texture.png)
+
+## 3D DOM
+
+If you want to take more of a general debugging view, rather than the z-index experience, the 3D DOM gives an overall look of the DOM. Since the z-index context is removed, the DOM is stacked more closely and cleanly. This pane has similar functionality, but there are a few nuances.
+
+### Changing your view
+
+In the **3D DOM** pane, the **Isolate selected element** filter has "Include children" and "Include parents." By default both checkboxes are selected, which means that clicking on the **Isolate selected element** button after selecting an element in the canvas would display the element chosen, the element's parents, and the element's children. Deselecting the "Include children" checkbox and clicking the **Isolate selected element** button again would display the selected element and the element's parents. If you select the "Include children" checkbox and deselect the "Include parents" checkbox before clicking on **Isolate selected element**, the canvas will then display the elemen and its children. If you deselect both checkboxes and click on **Isolate selected element**, the canvas will only display the element you previously selected.
+
+You'll notice a slider in the control pane titled **Nesting level** with a number next to it. The number indicates the number of layers in the document. Dragging the slider to the left will cause the outermost layers to peel away until you are left with a nesting level of 1, displaying only the furthest back element in the DOM. This allows you to remove some of the clutter if you are trying to get a closer look at what is going on in the lower levels.
+
+### DOM color type
+
+You'll notice that, in addition to the *Purple to White*, *Blue to Yellow*, *Rainbow*, and *Use background color* options, there is *Use screen texture*. The screen texture adds contect to your debugging experience by displaying the content from the web page directly onto the elements. This is still a work in progress, but feel free to send us feedback with the smiley to show us websites that have a harder time rendering their screen texture in the 3D View. 
+
+## Next steps
+
+We are working on the UI and adding more functionality to the 3D View based on asks from users like you. If you have an idea, let us know by clicking on the smiley in the browser or [tweeting](https://twitter.com/EdgeDevTools) us. If you want to be the first to know what's next, make sure to follow us, too. We can't wait to hear what you think.
```

Comment:
Again link to the feedback section in our Chromium doc. For tweeting, you can actually add an intent like this: https://twitter.com/intent/tweet?text=@EdgeDevTools which might make more sense in this context.